packages/terra-form/src/Control.jsx

```diff
@@ -82,16 +84,16 @@ const Control = ({
   value,
   ...customProps
 }) => {
-  const labelClassNames = classNames(
-    'terra-Form-control',
-    { 'terra-Form-control--inline': isInline },
+  const labelClassNames = cx([
+    'control',
+    { 'control-inline': isInline },
     customProps.className,
-  );
+  ]);
 
-  const labelTextClasses = classNames(
-    'terra-Form-control-label-text',
+  const labelTextClasses = cx([
+    'control-label-text',
     labelTextAttrs.className,
-  );
+  ]);
 
   const controlInputAttrs = Object.assign({}, inputAttrs);
```

Comment:
In the control.scss file, you changed `.terra-Form-input` to `.input` that is locally scoped to the control component. Do we need to add `className={cx('input')}` to the `controlInputAttrs`?